packages/forma-36-website/src/pages/components/example/index.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Example Component'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/FormLabel'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-formlabel--default'
+---
+
+Add a description about the component and its function in the user interface here
```

Comment:
Introduce the component with a description about the component and its function within the user interface.